client/tests/integration/smartcontracts/create_nft_for_every_user_trigger/src/lib.rs

```diff
@@ -43,7 +43,9 @@ fn main(_id: TriggerId, _owner: AccountId, _event: EventBox) {
         )
         .parse()
         .dbg_unwrap();
-        metadata.insert_with_limits(name, true, limits).dbg_unwrap();
+        metadata
+            .insert_with_limits(name, JsonString::new(&true), limits)
```

Comment:
```suggestion
            .insert_with_limits(name, JsonString::new(&true), limits)
```

the constructor shouldn't take a reference to a `Copy` type